internal/turso/turso.go

```diff
@@ -79,6 +82,13 @@ func (t *Client) newRequest(method, urlPath string, body io.Reader) (*http.Reque
 
 func (t *Client) do(method, path string, body io.Reader) (*http.Response, error) {
 	req, err := t.newRequest(method, path, body)
+	if flags.Debug() {
+		dump, err := httputil.DumpRequestOut(req, true)
+		fmt.Printf("%s", string(dump))
```

Comment:
```suggestion
		fmt.Println(string(dump))
```

Or just `fmt.Print` if you don't want the extra line break.